src/openrct2-ui/drawing/engines/opengl/DrawCommands.h

```diff
@@ -114,9 +114,9 @@ struct DrawRectCommand
 
     enum
     {
-        FLAG_NO_TEXTURE = (1 << 2),
-        FLAG_MASK = (1 << 3),
-        FLAG_CROSS_HATCH = (1 << 4),
+        FLAG_NO_TEXTURE = (1U << 2U),
```

Comment:
This is a minor detail, but we always write `1u` etc. for unsigned values.

Reply:
Certainly not, as there are no less than 781 instances of `1U` in the codebase. (778 of which are `1UL`.)

Reply:
Hm, in that case I think we should pick one, and apply it throughout. That is something for another PR, though.